core/circuitbreaker/circuit_breaker.go

```diff
@@ -237,8 +237,9 @@ func newSlowRtCircuitBreakerWithStat(r *Rule, stat *slowRequestLeapArray) *slowR
 
 func newSlowRtCircuitBreaker(r *Rule) (*slowRtCircuitBreaker, error) {
 	interval := r.StatIntervalMs
+	bucketCount := r.getValidStatSlidingWindowBucketCount()
```

Comment:
Use an independent func to calculate bucketCount.
Member function of Rule is not good

Reply:
Updated.